src/Sentry.Unity.Editor/ConfigurationWindow/AdvancedTab.cs

```diff
@@ -47,6 +47,16 @@ internal static void Display(ScriptableSentryUnityOptions options, SentryCliOpti
             EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
             EditorGUILayout.Space();
 
+            {
+                options.CaptureFailedRequests = EditorGUILayout.Toggle(
+                    new GUIContent("Capture Failed Requests", "Whether the SDK should capture failed HTTP requests."),
```

Comment:
Worth adding HTTP to be clear:

```suggestion
                    new GUIContent("Capture Failed HTTP Requests", "Whether the SDK should capture failed HTTP requests."),
```